src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+
+const Navbar = () => {
+  return (
+    <nav className="navbar flex">
+      <div className>
```

Comment:
You have left an empty `className` attribute, might have forgotten to add the classes, else if not in use then cleanup

Reply:
done! Thanks for pointing out.